### DISCUSSION TITLE
Feature request: Show pixel grid after zooming in

### DISCUSSION THREAD

**Discussion by Manueljlin** [uxbox/uxbox#519] [Ideas]
![image](https://user-images.githubusercontent.com/11275573/106599408-4c357a80-6559-11eb-92a9-91a28b2e5252.png)

Figma:
![image](https://user-images.githubusercontent.com/11275573/106599737-c6fe9580-6559-11eb-8031-c83bbde199f2.png)
![image](https://user-images.githubusercontent.com/11275573/106599762-ce25a380-6559-11eb-9a9f-94832038a834.png)

**Comment by elhombretecla** [comment]
Definitely!
Pixel grid would be a very useful feature. We add it in our list of enhancements.
